The_Glowing_Volcano/Assets/Scripts/MenuSound.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+[RequireComponent(typeof(Button))]
+public class MenuSound : MonoBehaviour
+{
+    public AudioClip sound; 
+
+    private Button button { get { return GetComponent <Button>(); } }
```

Comment:
This is an interesting application of properties! Good job using them! One thing to remember is that `GetComponent` an be expensive to call, so it is often a good idea to call it once in `Start` or `Awake` and "cache" the result in a variable so you don't have to call it every time the property is accessed as you do in your code here.

Also, properties should use PascalCase naming convention, not camelCase (even private ones)